index.php

```diff
@@ -118,6 +121,28 @@
 
 <p>Older versions of the OpenVX specification provided for reference.</p>
 
+<p><b>OpenVX 1.2</b></p>
+<ul>
+  <li>OpenVX 1.2 specification
```

Comment:
This appears to have omitted the 1.2.1 specification update (ISTM better to just link to 1.2.1 and not 1.2 as well).

Reply:
I agree that we need to keep the 1.2.1 version in the registry, but I think we should also keep the link to the "Previous Tools" version for reference.  We have been keeping all of our old specs in the "Older versions" section and I think it's good to have them available.